app/helpers/import_rooms.rb

```diff
@@ -0,0 +1,30 @@
+require 'csv'
+require 'point'
+require 'room'
+require 'building'
+require 'polyline'
+
+rooms = %w[A B C D E F G H I]
+building = Building.create
+
+csv_text = File.read('app/assets/data/importBuilding1.csv')
+csv = CSV.parse(csv_text)
+points = {}
```

Comment:
You say in the readme that you can just add more .csv files, but I'm not really seeing how that would look like in this file. Maybe you could create an array of csv files (containing only importBuilding1.csv for now) and then do the point import code for each csv in that array?

Pseudo code:
``` ruby
csv_paths = ['app/.../import1.csv']
for each csv_path in csv_paths do
   csv = CSV.parse(csv_path)
   rooms = %w...
   building = Building.create
   ...
end
```

Reply:
For me it would be fine, if the script can only read one csv file. You dont save that much time by that if you have to label the indoor maps.  
What do you think @sm1lla ?.

Reply:
I agree. What we meant was that you can import one csv file at a time and habe to change the path for every new csv file. I'm gonna change that in the readme.

Reply:
Okay, that also works